core/lib/Thelia/Controller/Admin/AdminLogsController.php

```diff
@@ -41,11 +43,12 @@ public function loadLoggerAjaxAction()
         ) as $entry) {
             $entries[] = array(
                 "head" => sprintf(
-                    "[%s][%s][%s:%s]",
+                    "[%s][%s][%s:%s%s]",
```

Comment:
Because it can be empty. So the colon is added in the argument when it 's not empty.